### PR TITLE
Enable beta channel desktop release

### PR DESCRIPTION
**Pull request by michalchecinski**
## Type of change

```
- [ ] Bug fix
- [ ] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [x] Build/deploy pipeline (DevOps)
- [ ] Other
```

## Objective

Add workflow to build and publish beta desktop releases to S3 bucket.

## Code changes

- **.github/workflows/release-desktop-beta.yml:** Test mobile release

## Before you submit

```
- [ ] I have checked for **linting** errors (`npm run lint`) (required)
- [ ] I have added **unit tests** where it makes sense to do so (encouraged but not required)
- [ ] This change requires a **documentation update** (notify the documentation team)
- [ ] This change has particular **deployment requirements** (notify the DevOps team)
```

### CODE REVIEW

**Comment by Hinton** [comment]
@michalchecinski looks like you accidentally re-added the jslib submodules. Please delete those folders.

File: .github/workflows/release-desktop-beta.yml [hunk 1]
```diff
@@ -70,18 +71,45 @@ jobs:
...
+          ver=${VERSION//$find/$replace}
+          branch_name=mobile-beta-release-$ver-beta
+
+          git switch -c $branch_name
+          git add .
+          git commit -m "Bump mobile version to $VERSION-beta"
+
+          git push -u origin $branch_name
+
+          echo "::set-output name=branch-name::$branch_name"
```
Comment (joseph-flinn) [inline .github/workflows/release-desktop-beta.yml:97]: Can we remove the mobile specific code here and exchange it for desktop?

**Review by joseph-flinn — CHANGES_REQUESTED** [review]
Could you remove the `jslib` submodules? I don't think we use the git submodule pattern anymore.

File: .github/workflows/release-desktop-beta.yml (hunk 1 above)
Comment (michalchecinski) [reply .github/workflows/release-desktop-beta.yml]: FIxed :)

**Review by joseph-flinn — COMMENTED** [review]
It looks like the linter is unhappy with the newest changes.

**Pull request merged**